dor/providers/operations.py

```diff
@@ -53,10 +52,14 @@ def run(self) -> None:
         except JHOVEDocError:
             return None
 
+        use_functions = [UseFunction.source]
+        if source_tech_metadata.mimetype == Mimetype.JP2:
+            use_functions.append(UseFunction.service)
```

Comment:
@ssciolla This will work for the JPEG2000s we currently get (page scans from vendors) that never need to be rotated. Otherwise, this should probably happen in the compress operation as an early exit because the source file would not be the access file.

Reply:
Okay, should I change something now?

Reply:
Move it to the compress operation; that's responsible for the service image and would be technically more correct.

Reply:
Ehhh. Add a "TODO" before you calculate the file path to revisit this after the conversation with DigiPres and we get guidance if we'll keep usig our file naming scheme.

Reply:
Otherwise, looks good.

Reply:
Done. https://github.com/mlibrary/dor-py/pull/81/commits/a73741ad8b1df55002740e3f40e30c65c9c651b8